app/serializers/rest/quote_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class REST::QuoteSerializer < ActiveModel::Serializer
+  attributes :state
```

Comment:
Perhaps this state should include "deleted" and "filtered" based on logic below.